pkg/clusterbuilder/providers/capv/values/cluster_values.yaml

```diff
@@ -1,6 +1,7 @@
 global:
   metadata:
     description: "E2E Test cluster"
+    description: "e2e-test-cluster"
```

Comment:
Duplicates? :/

Reply:
i don't know what you're talking about :angel: